install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -898,6 +898,25 @@ hubble:
     # port 80 if not.
     # servicePort: 80
 
+    # -- Enable prometheus metrics for cilium-operator on the configured port at
```

Comment:
This doc comment mentions `cilium-operator`, I believe it should be `hubble-relay`